examples/specs/geo_point.vl.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "width": 500,
+  "height": 300,
+  "projection": {
+    "type": "albersUsa"
+  },
+  "data": {
+    "url": "data/zipcodes.csv",
+    "format": {
+      "type": "csv"
+    }
+  },
+  "transform": [{"calculate": "substring(datum.zip_code, 0, 1)", "as": "digit"}],
+  "mark": "circle",
+  "encoding": {
+    "x": {
+      "field": "longitude",
```

Comment:
having both field and type represent the same thing is a bit redundant and verbose-- I wonder if we can offer a shorthand syntax (and automatically augment once of field or type)?